frontend/src/api/useAccessReview.ts

```diff
@@ -29,17 +29,29 @@ const checkAccess = ({
       },
     },
   };
-  return k8sCreateResource<SelfSubjectAccessReviewKind>({
-    model: SelfSubjectAccessReviewModel,
-    resource: selfSubjectAccessReview,
-  });
+  return (
+    k8sCreateResource<SelfSubjectAccessReviewKind>({
+      model: SelfSubjectAccessReviewModel,
+      resource: selfSubjectAccessReview,
+    })
+      // TODO: TypeScript doesn't realize this can be inferred as this type and thinks it is boolean[]
+      .then((result): [allowed: boolean, loaded: boolean] => [
+        result.status ? result.status.allowed : true,
+        true,
+      ])
+      .catch((e) => {
+        // eslint-disable-next-line no-console
+        console.warn('SelfSubjectAccessReview failed', e);
+        return [true, true]; // if it critically fails, don't block SSAR checks; let it fail/succeed on future calls
+      })
+  );
 };
 
 export const useAccessReview = (
```

Comment:
Do we remove this? Use the cache system for everything? Is this wise?

Reply:
maybe we can set it as deprecated in favor of the new cache system. I guess if you wanted to live outside the context of the cache system then this would be used